### PR TITLE
Split client stuff

### PR DESCRIPTION
**Pull request by cryogen**
Split the bundle and have the non changing dependencies in their own bundle which can be cached leaving the changing client smaller

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,8 +35,10 @@
     "cookie-parser": "^1.4.3",
     "cross-env": "^3.2.4",
     "css-loader": "^0.27.3",
+    "event-source-polyfill": "0.0.11",
```
Comment (ystros) [inline package.json]: It's not clear what changes necessitated adding this dependency.

Comment (cryogen) [reply package.json]: It's always technically been a dependency to support webpack on ie.  For all intents and purposes it's been pointless but the config i copied it from uses it so I left it in

File: package.json [hunk 2]
```diff
@@ -4,9 +4,8 @@
   "description": "A Game of Throne Second Edition Card Game Server",
   "main": "index.js",
   "scripts": {
-    "test": "npm run jasmine && npm run karma",
+    "test": "npm run jasmine",
     "jasmine": "cross-env JASMINE_CONFIG_PATH=./jasmine.json jasmine",
-    "karma": "cross-env karma start karma.config.js --single-run",
     "cover": "cross-env JASMINE_CONFIG_PATH=./jasmine.json istanbul cover --include-all-sources jasmine",
     "lint": "eslint server/ test/server/ client/ --ext=js --ext=jsx",
     "build": "cross-env NODE_ENV=production webpack -p --config webpack.config.production.js"
```
[the remark this answers is not in the document]
Comment (cryogen) [reply package.json]: Good catch, forgot about that

File: server/server.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ const Raven = require('raven');
 const webpackDevMiddleware = require('webpack-dev-middleware');
 const webpackHotMiddleware = require('webpack-hot-middleware');
 const webpack = require('webpack');
-const webpackConfig = require('../webpack.config.js');
+const webpackConfig = require('../webpack.config.js')();
```
[the remark this answers is not in the document]
Comment (cryogen) [reply server/server.js]: The parameter is magic webpack passes in when you run it in production mode

**Comment by cryogen** [comment]
I'll fix jenkins after its merged so it still runs for other PRs/commits

**Pull request merged**